en/08_Changelogs/5.4.0.md

```diff
@@ -261,6 +261,16 @@ The `SilverStripe\CMS\Model\SiteTree.DESCRIPTION` localisation key (along with t
 
 This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
 
+### Change to error logging
+
+Some errors were incorrectly being logged using the error handler service, which resulted in displaying the error in the browser and CLI and, in live mode, not displaying the rest of the response to users.
+
+This was the result of a misunderstanding about the difference between the `Psr\Log\LoggerInterface.errorhandler` error handler service and the `Psr\Log\LoggerInterface.errorhandler` logging service.
```

Comment:
```suggestion
This was the result of a misunderstanding about the difference between the `Psr\Log\LoggerInterface` error handler service and the `Psr\Log\LoggerInterface.errorhandler` logging service.
```
Or the other way around

Reply:
Oops yup. Other way around but good catch.